src/app/commands/admin/Account.ts

```diff
@@ -0,0 +1,54 @@
+import { CommandContext, CommandOptionType, SlashCommand, SlashCreator } from 'slash-create';
+import { LogUtils } from '../../utils/Log';
+import VerifyTwitter from '../../service/account/VerifyTwitter';
+import ServiceUtils from '../../utils/ServiceUtils';
+import discordServerIds from '../../service/constants/discordServerIds';
+
+export default class Account extends SlashCommand {
+	constructor(creator: SlashCreator) {
+		super(creator, {
+			name: 'account',
+			description: 'Manage your accounts integration.',
```

Comment:
Can we do `Manage your account's integration.`?

Reply:
done